src/tf_container/serve.py

```diff
@@ -134,6 +139,10 @@ def _wait_model_to_load(grpc_proxy_client, max_seconds):
     raise ValueError(message.format(max_seconds))
 
 
+def _get_first_safe_port(port_range):
```

Comment:
1. should be in sagemaker-containers
2. should take optional 2nd arg for max port already used... e.g.
```
def next_safe_port(port_range, after=None):
  # returns first safe port or after+1, 
  # raises ValueError if after+1 > max port
```

Reply:
https://github.com/aws/sagemaker-containers/pull/128